spec/lib/tasks/scripts/soft_delete_duplicate_patients_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+require 'tasks/scripts/soft_delete_duplicate_patients'
+
+RSpec.describe SoftDeleteDuplicatePatients do
+  let!(:file_path) { 'spec/fixtures/files/patient_de_dup_test.csv' }
```

Comment:
We should be able to make most if not all of these simple `let`s instead of `let!`s. They can be lazily evaluated.